package-lock.json

```diff
@@ -7916,14 +7915,12 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
Not sure what update this refers to but should this be in the commit?

Reply:
I know it's an unsatisfying answer, but this is just what I get from running `npm install inputmask`. 🤷‍♂